kubetest2-gke/deployer/deployer.go

```diff
@@ -169,7 +175,7 @@ func New(opts types.Options) (types.Deployer, *pflag.FlagSet) {
 			WindowsMachineType: defaultWindowsNodePool.MachineType,
 			WindowsImageType:   defaultWindowsImage,
 			// Leave Version as empty to use the default cluster version.
```

Comment:
Comment also needs to be updated to `ClusterVersion`

Reply:
+1

Reply:
done